src/Providers/ApartmentServiceProvider.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace BuildEmpire\Apartment\Providers;
+
+use BuildEmpire\Apartment\Commands\ApartmentMakeCommand;
+use BuildEmpire\Apartment\Commands\ApartmentDropCommand;
+use BuildEmpire\Apartment\Commands\ApartmentListCommand;
+use BuildEmpire\Apartment\Commands\ApartmentMigrationCommand;
+use Illuminate\Support\ServiceProvider;
+use BuildEmpire\Apartment\Schema;
+
+class ApartmentServiceProvider extends ServiceProvider
+{
+    /**
+     * Register any application services.
+     *
+     * @return void
+     */
+    public function register()
+    {
+        if ($this->app->runningInConsole()) {
+            $this->commands([
+                ApartmentMakeCommand::class,
+                ApartmentDropCommand::class,
+                ApartmentListCommand::class,
+                ApartmentMigrationCommand::class,
+            ]);
+        }
+
+        $this->app->singleton('BuildEmpire\Apartment\Schema', function () {
+            return new Schema();
+        });
+    }
+
+    /**
+     * Boot any requirements.
+     */
+    public function boot()
+    {
+        $this->loadViewsFrom(__DIR__ . '/../Views', 'apartment');
```

Comment:
Knowing nothing about how laravel/php/computers work, I assume this doesn't actually expose the view for rendering on the consuming application?  It would need a route/controller configuring presumably for that to happen?  I doubt it would be a problem even if it did, but just a random thought that occurred to me.